charts/map/templates/secrets.yaml

```
@@ -0,0 +1,13 @@
+{{- if .Values.secrets.create }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "map.fullname" . }}
+  labels:
+    app: {{ template "map.fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+type: Opaque
+data:
+  next-public-map-api-token: {{ .Values.secrets.nextPublicMapApiToken | b64enc | quote }}
```

```suggestion
  map-api-key: {{ .Values.secrets.mapApiKey | b64enc | quote }}
```
We don't need the NEXT_PUBLIC prefix everywhere. It's only required by next, so it's redundant here and adds unnecessary verbosity.